bots/tests-dequeue-one

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+MAX_PRIORITY = 12
+
+import argparse
+import pika
+import subprocess
+import sys
+
+def main():
+    parser = argparse.ArgumentParser(description='Bot: read test commands from the queue and execute them')
+    parser.add_argument('--amqp', action="store_true", default='localhost',
+                        help='The url of the amqp server to consume from.')
+    parser.add_argument('--queue', action="store_true", default='tests',
+                        help='The name of the queue to consume from.')
```

Comment:
`store_true` is wrong for these, as they get a string argument, these are not boolean. `store` is the default, so just leave out `action`.